app/client/components/productSwitch/CancellationSwitchConfirmed.tsx

```diff
@@ -0,0 +1,274 @@
+					plan. Please check your inbox for an email containing all
+					your details and information on how to access your benefits.
+				</p>
+				<div
+					css={css`
+						border: 1px solid ${neutral[86]};
+					`}
+				>
+					<div
+						css={css`
+							background-color: #e3edfe;
+
+							padding: ${space[4]}px ${space[4]}px 0 ${space[4]}px;
+
+							${minWidth.tablet} {
+								display: flex;
+								justify-content: space-between;
+								padding: ${space[4]}px ${space[5]}px 0
+									${space[5]}px;
+							}
+						`}
+					>
+						<p
+							css={css`
+								${headline.xsmall({ fontWeight: 'bold' })};
+							`}
+						>
+							What happens now?
+						</p>
+						<GridPicture
+							cssOverrides={css`
+								display: flex;
+								margin: auto;
+
+								${minWidth.tablet} {
+									display: block;
+									margin-right: 42px;
+								}
+
+								max-width: 220px;
+							`}
+							sources={[
+								{
+									gridId: 'digitalSubPackshot',
+									srcSizes: [497, 285],
+									imgType: 'png',
+									sizes: '100vw',
+									media: '(max-width: 220px)',
+								},
+							]}
+							fallback="digitalSubPackshot"
+							fallbackSize={497}
+							altText=""
+							fallbackImgType="png"
+						/>
+					</div>
+
+					<div
+						css={css`
+							padding: ${space[6]}px ${space[5]}px;
+							max-width: 60ch;
+						`}
+					>
+						<ul css={listCss}>
+							<li>
+								Check your inbox for an email containing all
+								your important information, plus details on how
+								to access your benefits.
+							</li>
+							<li>
+								We'll stop your{' '}
+								{newProduct.billing.frequency.name}ly{' '}
+								{productSwitchContext.productType.friendlyName}{' '}
+								payments.
+							</li>
+							{newProduct && (
+								<>
+									<li>
+										Your new {newProduct.name} starts today.
+									</li>
+									<li>
+										Your first payment of{' '}
+										{productFirstPaymentAmount(newProduct)}{' '}
+										will be taken on{' '}
+										{productStartDate(newProduct)}.
+									</li>
+								</>
+							)}
+							<li>
+								You can manage your subscription online from
+								your{' '}
+								<Link
+									to="/"
+									css={css`
+										color: ${brand[500]};
+										text-decoration: underline;
+									`}
+								>
+									Account overview.
+								</Link>
+							</li>
+						</ul>
+					</div>
+				</div>
+
+				<div
+					css={css`
+						border: 1px solid ${neutral[86]};
+					`}
+				>
+					<div
+						css={css`
+							display: flex;
+							justify-content: space-between;
+							align-items: start;
+							background-color: ${brand[400]};
+
+							${minWidth.mobileLandscape} {
+								align-items: center;
+							}
+						`}
+					>
+						<h2
+							css={css`
+								${headline.xsmall({ fontWeight: 'bold' })};
+								margin: 0;
+								padding: ${space[3]}px 20px;
```

Comment:
padding: ${space[3]}px ${space[5]}px;

Reply:
Good spot. I've updated a few spacing values with the Source equivalent, but there may be one or two still lurking…